test/basic_SUITE.erl

```diff
@@ -32,7 +32,7 @@ all() ->
      epochs_gc,
      callback_message,
      self_callback_message,
-     epoch_no_state,
+     %epoch_no_state,
```

Comment:
is this test obsolete?  might as well delete it if so.  otherwise we should re-enable and fix if needed.